apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.ts

```diff
@@ -13,23 +12,28 @@ import * as opsbot from '../../lib/opsbot';
 import * as github from '../../lib/github';
 import { config } from '../../lib/config';
 import { CourseRequestSchema, IdSchema } from '../../lib/db-types';
+import { RequestCourse } from './instructorRequestCourse.html';
 
 const router = express.Router();
 const sql = loadSqlEquiv(__filename);
 
 router.get(
   '/',
   asyncHandler(async (req, res) => {
-    res.locals.navPage = 'request_course';
-    res.locals.course_requests = await queryRows(
+    const course_requests = await queryRows(
       sql.get_requests,
       {
         user_id: res.locals.authn_user.user_id,
       },
```

Comment:
This is really a nit, but it bothers me that Prettier does not resolve this without a nudge.
```suggestion
      { user_id: res.locals.authn_user.user_id },
```

Reply:
I've given up trying to fight with the formatters. 😸